LamestWebserver/LamestWebserver/Core/Logger.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+
+namespace LamestWebserver.Core
+{
+    /// <summary>
+    /// A simple Logger for LamestWebserver.
+    /// </summary>
+    public class Logger
+    {
```

Comment:
also probably switch all the > to <, and swap the logging level numbers in the enum